x-pack/plugins/lens/public/indexpattern_datasource/dimension_panel/field_input.tsx

```diff
@@ -62,7 +62,7 @@ export function FieldInput({
       <FieldSelect
         fieldIsInvalid={currentFieldIsInvalid}
         currentIndexPattern={indexPattern}
-        existingFields={existingFields}
+        existingFields={existingFields[indexPattern.title]}
```

Comment:
```suggestion
        existingFields={existingFields[indexPattern.title]}
```
Otherwise, when choosing multiterms, the editor explodes.